.eslintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
-  extends: ["@stoplight"],
+  extends: ['@stoplight'],
+  rules: {
+    'no-param-reassign': 'off',
+  },
```

Comment:
Why such rule?

Reply:
For this - https://github.com/stoplightio/http-spec/pull/97/files#diff-c84e068125096414207643df176f27fbR17

I suppose I could ignore at the line level if you prefer?

Reply:
I cannot identify the line